test/tests/functional/pbs_svr_dyn_res.py

```diff
@@ -480,3 +480,237 @@ def test_res_size_runtime(self):
         job_comment += " foobar (R: 95gb A: 50gb T: 50gb)"
         a = {'job_state': 'Q', 'comment': job_comment}
         self.server.expect(JOB, a, id=jid, attrop=PTL_AND)
+
+    def test_res_size_invalid_input(self):
+        """
+        Test invalid values returned from server_dyn_resource
+        script for resource type 'size'.
+        Script returns a 'string' instead of type 'size'.
+        """
+        # Create a resource of type size
+        resname = ["foobar"]
+        restype = ["size"]
+        resflag = ["q"]
```

Comment:
server_dyn_res resources are not supposed to have flag=q.  This will cause the resource to be double counted.  It doesn't really matter for this test, but we might as well set things up right.

Reply:
Removed flag